File: RefHammer/App/refhammer/src/app/Models/factions.ts
export class Faction {
    id: number;
    abbreviation: string;
    name: string;
    link: string;
    isSubfaction: string;
    parentId: string;
  
    constructor(id: number, abbreviation: string, name: string, link: string, isSubfaction: string, parentId: string) {
      this.id = id;
      this.abbreviation = abbreviation;
      this.name = name;
      this.link = link;
      this.isSubfaction = isSubfaction;
      this.parentId = parentId;
    }
  }

File: RefHammer/Model/Factions.cs
namespace RefHammer.Model
{
    public class Factions
    {
        public int Id { get; set; }
        public string Abbreviation { get; set; }
        public string Name { get; set; }
        public string Link { get; set; }
        public string IsSubfaction { get; set; }
        public string ParentId { get; set; }

    }
}
